src/main/java/emissary/client/response/Directory.java

```diff
@@ -0,0 +1,64 @@
+package emissary.client.response;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.io.Serializable;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+
+@XmlAccessorType(XmlAccessType.NONE)
+public class Directory implements Comparable<Directory>, Serializable {
+
+    private static final long serialVersionUID = 2428511052308449193L;
+
+    @XmlElement(name = "dataId")
+    private String dataId;
+
+    @XmlElement(name = "entry")
+    private String entry;
+
+    public Directory() {}
+
+    public Directory(String dataId, String entry) {
```

Comment:
Maybe create a constructor that takes a [DirectoryEntry](https://github.com/NationalSecurityAgency/emissary/blob/master/src/main/java/emissary/directory/DirectoryEntry.java) in case we need more info.